File: app/src/main/java/com/coinninja/coinkeeper/util/Shuffler.java
package com.coinninja.coinkeeper.util;

import java.util.Collections;
import java.util.List;
import java.util.Random;

import javax.inject.Inject;

public class Shuffler {

    @Inject
    public Shuffler() {
    }

    public void shuffle(List items) {
        Collections.shuffle(items);
    }

    public int pick(int numChoices) {
        return new Random().nextInt(numChoices);

    }

}

File: app/src/main/java/com/coinninja/coinkeeper/util/Shuffler.kt
package com.coinninja.coinkeeper.util

import app.dropbit.annotations.Mockable
import java.util.*
import javax.inject.Inject

@Mockable
class Shuffler @Inject constructor() {
    fun shuffle(items: List<*>?) {
        Collections.shuffle(items)
    }

    fun pick(numChoices: Int): Int {
        return Random().nextInt(numChoices)
    }
}